src/Cake.Npm/BumpVersion/NpmBumpVersionSettingsExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Cake.Npm.BumpVersion
+{
+    using System;
+
+    /// <summary>
+    /// Extensions for <see cref="NpmBumpVersionSettings"/>.
+    /// </summary>
+    public static class NpmBumpVersionSettingsExtensions
+    {
+        /// <summary>
+        /// Defines that npm version should commit, even if repository is not clean
+        /// exists on disk.
```

Comment:
Something seems wrong with the second part of this sentence

Reply:
✔️